### ISSUE TITLE
MailHog not catching emails, site cannot send emails

### ISSUE THREAD

**Issue by drubage** [docksal/docksal#208]
I have created a Drupal project from scratch using `fin project create` and then followed [these steps](http://docksal.readthedocs.io/en/master/tools/mailhog/) to create a MailHog container but when I try to send a password reset email via Drupal 8 I get the error "Unable to send email. Contact the site administrator if the problem persists". FYI on Windows I also had to add a host entry for `192.168.64.100 webmail.drupal8test.docksal` to my windows hosts file to get the MailHog server to load in my browser (which it successfully does).

**Comment by achekulaev** [comment]
Can you make sure your `php.ini` got populated in the container?

```bash
fin exec cat /var/www/.docksal/etc/php/php.ini
```

Does it contain settings that should be there as per instruction?

**Comment by drubage** [comment]
I stopped the VM and re-started it then re-initialized the project and it worked. Something about the `fin up` didn't seem to register something when I added mailhog but I guess having it in the project from the start works okay. I didn't get to check the php setting as I had to re-start the VM but I am pretty sure I looked at the status page for the drupal site and it did have the right php setting at the time.
